src/RelatedLogs/RelatedLogsScene.tsx

```diff
@@ -133,9 +135,16 @@ export class RelatedLogsScene extends SceneObjectBase<RelatedLogsSceneState> {
     }
 
     // Set up UI for logs panel
-    const logsPanelContainer = sceneGraph.findByKeyAndType(this, LOGS_PANEL_CONTAINER_KEY, SceneCSSGridItem);
+    const logsPanelContainer = sceneGraph.findByKeyAndType(this, LOGS_PANEL_CONTAINER_KEY, SceneFlexItem);
     logsPanelContainer.setState({
-      body: PanelBuilders.logs().setTitle('Logs').setData(this._queryRunner).build(),
+      body: PanelBuilders.logs()
+        .setTitle('Logs')
```

Comment:
Here can be seen how we pass different callbacks to the visualization that enable interactions within log details:

https://github.com/grafana/logs-drilldown/blob/main/src/Components/ServiceScene/LogsPanelScene.tsx#L335

For example, what happens when these buttons are clicked:

<img width="54" height="43" alt="imagen" src="https://github.com/user-attachments/assets/b2d3aace-d169-4f07-896f-6dfbc05668c4" />